### PR TITLE
Nested packages in Swing UI

### PR DESCRIPTION
**Pull request by ramidzkh**
Some testing appreciated

image

### CODE REVIEW

File: enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java [hunk 1]
```diff
@@ -381,10 +382,11 @@ public void addSeparatedClasses(List<ClassEntry> obfClasses, List<ClassEntry> de
 				return;
 			}
 
-			ClassEntry deobfEntry = mapper.deobfuscate(entry);
+			TranslateResult<ClassEntry> result = mapper.extendedDeobfuscate(entry);
+			ClassEntry deobfEntry = result.getValue();
 
 			List<ObfuscationTestService> obfService = enigma.getServices().get(ObfuscationTestService.TYPE);
-			boolean obfuscated = deobfEntry.equals(entry);
+			boolean obfuscated = result.isObfuscated() && deobfEntry.equals(entry);
```
Comment (2xsaiko) [inline enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java:389]: Is the `deobfEntry.equals(entry)` still needed here?

**Review by 2xsaiko — COMMENTED** [review]
LGTM, haven't tested it though.

File: enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java (hunk 1 above)
Comment (ramidzkh) [reply enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java:389]: I don't think so, but I think it'd be best if these were cleaned up in another PR where classes marked deobfucated but with their obfuscated name aren't bugged in the Swing UI

**Review by modmuss50 — APPROVED** [review]
Tested works as expected. Thanks 👍

**Pull request merged**